src/ClientData/include/ClientData/CallstackData.h

```diff
@@ -38,19 +38,16 @@ class CallstackData {
   void AddCallstackFromKnownCallstackData(const orbit_client_protos::CallstackEvent& event,
                                           const CallstackData& known_callstack_data);
 
-  [[nodiscard]] const absl::flat_hash_map<int32_t,
-                                          std::map<uint64_t, orbit_client_protos::CallstackEvent>>&
-  callstack_events_by_tid() const {
-    return callstack_events_by_tid_;
+  [[nodiscard]] size_t callstack_events_by_tid_size() const {
```

Comment:
Let's just remove this method. It's only used for debugging and I'm not sure to what end. And with this name the purpose is unclear (number of tids with at least one event?).